vm/loader/manifests/openhcl-x64-direct-release.json

```diff
@@ -7,7 +7,7 @@
             "isolation_type": "none",
             "image": {
                 "openhcl": {
-                    "command_line": "OPENHCL_FORCE_LOAD_VTL0_IMAGE=linux OPENHCL_BOOT_LOG=com3",
+                    "command_line": "OPENHCL_FORCE_LOAD_VTL0_IMAGE=linux OPENHCL_BOOT_LOG=com3 OPENHCL_ENABLE_VTL2_GPA_POOL=debug",
```

Comment:
Should we instead default to release unless you specify debug? One issue (this is a release json) and really, debug here means "CI test" so should we override it to that? 

The rationale being that folks generally do use debug builds in tests, and we probably want the "prod" logic to takeover. It seems to me the other buckets are more for _ci_ not debug?